python_ta/debug/accumulation_table.py

```diff
@@ -102,9 +101,17 @@ def _create_iteration_dict(self) -> dict:
         and loop variable to its respective value during each iteration
         """
         return {
-            "iteration": list(range(len(self.loop_var_val))),
-            "loop variable (" + self.loop_var_name + ")": self.loop_var_val,
-            **self.loop_accumulators,
+            "iteration": list(range(len(list(self.loop_variables.values())[0]))),
+            **{
+                loop_var.replace(loop_var, "loop variable (" + loop_var + ")"): loop_vals
```

Comment:
Okay I did some thinking about this and I'm worried that the headers will make the table unnecessarily wide. I'm also not sure we need to draw a distinction between "loop variables" and "accumulators".

So let's actually change the headings so they just show the variable name, and no "loop variable" or "accumulator" labels. This will simplify the code as well.

Please make sure to update the example in the documentation page after making this change. :)